### ISSUE TITLE
How do I keep slides static? If I want users to navigate the slide themselves without the "slideDuration" how would I go about doing that? 

### ISSUE THREAD

**Issue by tarmacjupiter** [AJHenry/react-google-slides#5]

**Comment by AJHenry** [comment]
Hey, as of now this is not possible

I will add this functionality later today

**Comment by tarmacjupiter** [comment]
Cool! Thanks so much!

**Comment by AJHenry** [comment]
Fixed this with `3.0.0`, excluding slide duration will now require the user to manually go through the slides

```sh
npm i react-google-slides@^3.0.0
yarn add react-google-slides@^3.0.0
```

Thanks!
